### PR TITLE
Create Dockerfiles for cassandra, cassandra-flask-server and nginx-rtmp

### PR DESCRIPTION
**Pull request by chc5**

### CODE REVIEW

File: src/db/cass/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+WORKDIR /cass-app
+
+ENV NGINX_RTMP_IP=127.0.0.1
+
+ENV NGINX_RTMP_PORT=19350
+
+ENV CASSANDRA_CLUSTER_IPS="192.168.1.15"
+
+ENV CASSANDRA_CLUSTER_PORT="9043"
```
Comment (yliu0715) [inline src/db/cass/Dockerfile:11]: I think you meant `9042`?

File: src/db/cass/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+ENV CERT_FILE_ADDRESS=192.168.1.15:8008/meowl_nginx.crt
+
+ENV FLASK_APP=/cass-app/app.py
+
+RUN apt-get update
+
+RUN apt-get -y install sudo
```
Comment (yliu0715) [inline src/db/cass/Dockerfile:19]: `sudo apt-get -y install` ?

File: src/db/cass/Dockerfile (hunk 1 above)
Comment (chc5) [reply src/db/cass/Dockerfile:9]: I'm just testing it so it could be connected to any port.

File: src/db/cass/Dockerfile (hunk 2 above)
Comment (chc5) [reply src/db/cass/Dockerfile:16]: What about it

Comment (yliu0715) [reply src/db/cass/Dockerfile]: Shouldn't it be `sudo apt-get -y install` and not `apt-get -y install sudo` ?

Comment (yliu0715) [reply src/db/cass/Dockerfile]: Ahh nevermind. you're installing sudo

Comment (melvinheks) [reply src/db/cass/Dockerfile]: sudo install sudo

Comment (chc5) [reply src/db/cass/Dockerfile]: yes install yes

File: src/db/cass/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,40 @@
+FROM python:3.6
+
+WORKDIR /cass-app
+
+# Sets environment variables on the server
+ENV NGINX_RTMP_IP=127.0.0.1
+ENV NGINX_RTMP_PORT=19350
+ENV CASSANDRA_CLUSTER_IPS="192.168.1.15"
```
Comment (yliu0715) [inline src/db/cass/Dockerfile:8]: I'm assuming this is just a placeholder ip right now?

Comment (yliu0715) [reply src/db/cass/Dockerfile:8]: aaaaa

Comment (chc5) [reply src/db/cass/Dockerfile:8]: That's where your machine IP is on the local network. Try to use localhost.

Comment (yliu0715) [reply src/db/cass/Dockerfile:8]: yes

Comment (yliu0715) [reply src/db/cass/Dockerfile:8]: hello

Comment (yliu0715) [reply src/db/cass/Dockerfile:8]: I got these 2 errors about changing to localhost

Comment (chc5) [reply src/db/cass/Dockerfile:8]: You need to run these in order: 
nginx-rtmp
cassandra
cass-app

Comment (chc5) [reply src/db/cass/Dockerfile:8]: I take it back I forgot to mention that 127.0.0.1 points to the docker instance and not the host instance. So you need to route it via local WiFi network.

File: src/db/cass/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,41 @@
+FROM python:3.6
+
+WORKDIR /cass-app
+
+# Sets environment variables on the server
+ENV NGINX_RTMP_IP=${NGINX_RTMP_IP:-127.0.0.1}
+ENV NGINX_RTMP_PORT=${NGINX_RTMP_PORT:-19350}
```
Comment (melvinheks) [inline src/db/cass/Dockerfile:7]: Shouldnt this be 8000?

**Pull request merged**